status/profile_plots.py

```diff
@@ -87,7 +87,8 @@ def generate_profile_plot(x, y, z, cmap, title='Glider Profiles', ylabel='Pressu
     vmax = mean + 2 * std
     if vmin < 0:
         vmin = 0
-    im = ax.pcolormesh(x, y, z, cmap=cmap, vmin=vmin, vmax=vmax)
+    im = ax.scatter(x, y, c=z, cmap=cmap, vmin=vmin, vmax=vmax)
+    # im = ax.contourf(x, y, z, cmap=cmap, vmin=vmin, vmax=vmax)
```

Comment:
Can you remove these commented code snippets entirely?  I got confused last time about dead code that wasn't meant to be run.